src/service/db/dashboards/mod.rs

```diff
@@ -32,52 +34,7 @@ pub(crate) async fn get(
 ) -> Result<Dashboard, anyhow::Error> {
     let key = format!("/dashboard/{org_id}/{folder}/{dashboard_id}");
     let bytes = db::get(&key).await?;
-    let dash_str = std::str::from_utf8(&bytes)?;
-    let hash = config::utils::hash::gxhash::new()
-        .sum64(dash_str)
-        .to_string();
-    let d_version: DashboardVersion = json::from_slice(&bytes)?;
-    if d_version.version == 1 {
-        let dash: v1::Dashboard = json::from_slice(&bytes)?;
-        Ok(Dashboard {
-            v1: Some(dash),
-            version: 1,
-            hash,
-            ..Default::default()
-        })
-    } else if d_version.version == 2 {
-        let dash: v2::Dashboard = json::from_slice(&bytes)?;
-        Ok(Dashboard {
-            v2: Some(dash),
-            version: 2,
-            hash,
-            ..Default::default()
-        })
-    } else if d_version.version == 3 {
-        let dash: v3::Dashboard = json::from_slice(&bytes)?;
-        Ok(Dashboard {
-            v3: Some(dash),
-            version: 3,
-            hash,
-            ..Default::default()
-        })
-    } else if d_version.version == 4 {
-        let dash: v4::Dashboard = json::from_slice(&bytes)?;
-        Ok(Dashboard {
-            v4: Some(dash),
-            version: 4,
-            hash,
-            ..Default::default()
-        })
-    } else {
-        let dash: v5::Dashboard = json::from_slice(&bytes)?;
-        Ok(Dashboard {
-            v5: Some(dash),
-            version: 5,
-            hash,
-            ..Default::default()
-        })
-    }
+    from_json(&bytes)
```

Comment:
I just moved this logic into its own `from_json` function since we use the same logic in multiple places.